Composer/packages/ui-plugins/virtual-assistant-creation/src/components/configSummaryPage.tsx

```diff
@@ -0,0 +1,89 @@
+/* eslint-disable format-message/literal-pattern */
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+/** @jsx jsx */
+import { jsx, css } from '@emotion/core';
+import formatMessage from 'format-message';
+import React, { Fragment, useContext } from 'react';
+import { RouteComponentProps } from '@reach/router';
+import { DialogWrapper, DialogTypes } from '@bfc/ui-shared';
+import { Text } from 'office-ui-fabric-react/lib/Text';
+import { Separator, ISeparatorStyles } from 'office-ui-fabric-react/lib/Separator';
+
+import { RouterPaths } from '../shared/constants';
+
+import { AppContext } from './VirtualAssistantCreationModal';
+import { DialogFooterWrapper } from './dialogFooterWrapper';
+
+// -------------------- Styles -------------------- //
+
+const categoryTextStyling = css`
+  display: block;
+  margin-top: 10px;
+`;
+
+const entryText = (bold: boolean) => css`
+  display: inline;
+  font-weight: ${bold ? 'bold' : 'initial'};
+`;
+
+const separatorStyles: Partial<ISeparatorStyles> = {
+  root: { color: 'black' },
+};
+
+// -------------------- ConfigSummaryPage -------------------- //
+type ConfigSummaryPageProps = {
+  onDismiss: () => void;
+} & RouteComponentProps<{}>;
+
+export const ConfigSummaryPage: React.FC<ConfigSummaryPageProps> = (props) => {
+  const { state } = useContext(AppContext);
+  const onDismiss = props.onDismiss;
+
+  const categoryText = (text: string) => {
```

Comment:
Prefer prefix of render (e.g renderCategoryText) to avoid confusion between data variable and rendering helper functions.